### PR TITLE
Employee calendar view

### PR DESCRIPTION
**Pull request by predrag-njegovanovic**

### CODE REVIEW

File: app/controllers/profiles/employee_calendar_controller.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+class Profiles::EmployeeCalendarController < ApplicationController
+
```
Comment (nemanja-m) [inline app/controllers/profiles/employee_calendar_controller.rb:2]: We need `before_action :authenticate_employee!` here.

File: app/controllers/employees/registrations_controller.rb [hunk 2]
```diff
@@ -18,8 +15,10 @@ def create
     password = generate_password
     params[:employee][:password] = password
     params[:employee][:password_confirmation] = password
-
-    super
+    super do |employee|
+      employee.save!
+    end
+    byebug
```
Comment (nemanja-m) [inline app/controllers/employees/registrations_controller.rb:21]: Bye byebug :D

File: app/controllers/home_page/employee_home_controller.rb [hunk 3]
```diff
@@ -5,6 +5,8 @@ class HomePage::EmployeeHomeController < ApplicationController
   layout "home_page"
 
   def index
+    @restauran = Restaurant.where(manager_id: current_employee.manager_id).first
+    @employee = Employee.where(:id => current_employee.id).first
```
Comment (nemanja-m) [inline app/controllers/home_page/employee_home_controller.rb:9]: :thinking: Do we need `@employee` ? `current_employee` is already set.

Comment (tamarakatic) [reply app/controllers/home_page/employee_home_controller.rb:9]: 😏

**Pull request closed without merging**